internal/wasm/sdk/test/e2e/external_test.go

```diff
@@ -111,15 +111,22 @@ func assert(t *testing.T, tc cases.TestCase, result *opa.Result, err error) {
 	t.Helper()
 	if tc.WantDefined != nil {
 		if err != nil {
-			t.Fatal("unexpected error:", err)
+			t.Fatalf("unexpected error: %v", err)
 		}
 		assertDefined(t, defined(*tc.WantDefined), result)
 	} else if tc.WantResult != nil {
 		if err != nil {
-			t.Fatal("unexpected error:", err)
+			t.Fatalf("unexpected error: %v", err)
 		}
 		assertResultSet(t, *tc.WantResult, tc.SortBindings, result)
 	} else if tc.WantErrorCode != nil || tc.WantError != nil {
+		if tc.StrictError {
+			if err != nil {
+				t.Fatalf("unexpected error: %v", err)
+			}
+			assertEmptyResultSet(t, result)
+			return
+		}
```

Comment:
It took me a minute to understand what's going on here--a short comment could be helpful. Something like "the wasm compiler does not support strict errors so if the error condition is only visible when strict errors are enabled, expect an empty/undefined result from evaluation."

Reply:
Thanks, I like that sentence. I'll use it. 😎